### DISCUSSION TITLE
Doxygen group

### DISCUSSION THREAD

**Discussion by jurgen-lentz** [michaeljones/breathe#982] [Q&A]
How can I create a reference to a doxygengroup?

**Comment by brtnfld** [comment]
I have a similar issue. The link to the Doxygen group is correct when I do doxygen->HTML, but when I use Breathe+Sphinx, the group description gets inserted at the link location, but there is no link back to the reference group. 

Have you found a workaround to the issue? 
I'm assuming that one will need to put an actual html reference instead of an `@ref`, which is not ideal.

**Reply by jurgen-lentz** [reply]
No, unfortunately not.
